jsx/Form.js

```diff
@@ -2549,7 +2549,7 @@ class RadioElement extends React.Component {
 
     const styleRow = {
       display: 'flex',
-      flexDirection: 'row',
+      flexDirection: this.props.vertical == false ? 'row' : 'column',
```

Comment:
```suggestion
      flexDirection: this.props.vertical ? 'column' : 'row',
```
This shouldn't cause any issues but the suggestion or '===' are more conventional in JS. Perhaps I am overlooking a detail which gives reasoning to the original decision.

The feature otherwise works great and as intended!